proto/message.py

```diff
@@ -375,7 +375,12 @@ def from_json(cls, payload, *, ignore_unknown_fields=False) -> "Message":
         return instance
 
     def to_dict(
-        cls, instance, *, use_integers_for_enums=True, preserving_proto_field_name=True
+        cls, 
+        instance, 
+        *, 
+        use_integers_for_enums=True, 
+        preserving_proto_field_name=True,
+        including_default_value_fields=True,
```

Comment:
The default for `MessageToDict` is `False`, and implicitly changing it here could constitute a breaking change. I'm okay with adding this option, but I think we should keep the default False.